### ISSUE TITLE
train loss 过大

### ISSUE THREAD

**Issue by MCdavidjames** [Ucas-HaoranWei/GOT-OCR2.0#316]
初始loss为5.几，按道理finetune的初始loss应该挺小的吧？
我是想先用模型试试单繁体字的训练，我看了看我的数据集应该没问题吧？
{
        "image": "images/7967_0.png",
        "conversations": [
            {
                "from": "human",
                "value": "<image>\nOCR:"
            },
            {
                "from": "gpt",
                "value": "左"
            }
        ]
    },
    {
        "image": "images/7967_1.png",
        "conversations": [
            {
                "from": "human",
                "value": "<image>\nOCR:"
            },
            {
                "from": "gpt",
                "value": "定"
            }
        ]
    },
然后下面是我的训练脚本
deepspeed GOT/train/train_GOT.py 
--deepspeed GOT-OCR2.0-main/GOT-OCR-2.0-master/zero_config/zero2.json 
--model_name_or_path GOT-OCR2.0-main/GOT-OCR-2.0-master/GOT_weights/ 
--use_im_start_end True 
--bf16 False 
--fp16 True 
--gradient_accumulation_steps 2 
--evaluation_strategy "no" 
--save_strategy "steps" 
--save_steps 200 
--save_total_limit 1 
--weight_decay 0. 
--warmup_ratio 0.001 
--lr_scheduler_type "cosine" 
--logging_steps 1 
--tf32 True 
--model_max_length 8192 
--gradient_checkpointing True 
--dataloader_num_workers 8 
--report_to none 
--per_device_train_batch_size 2 
--num_train_epochs 1 
--learning_rate 2e-5 
--datasets gj-ocr 
--output_dir /GOT-OCR2.0-main/GOT-OCR-2.0-master/out/simple/weight1
可以麻烦作者大大看看吗？

**Comment by Ucas-HaoranWei** [comment]
没有`<image>`?

**Comment by MCdavidjames** [comment]
![Image](https://github.com/user-attachments/assets/d84b8e87-be1d-4e91-8e0b-58942c065290)有的，好像打出来没了

> 没有?`<image>`
